js/components/post/list.vue

```diff
@@ -34,7 +34,7 @@ export default {
             },{
                 label: this._('Creation'),
                 key: 'created_at',
-                sort: 'created',
+                sort: 'created_at',
```

Comment:
Discussed with @taniki : created needed to be changed into created_at so that the fix below could work.